### DISCUSSION TITLE
Clarification on Building Upon Existing Work vs. Starting From Scratch

### DISCUSSION THREAD

**Discussion by KYash03** [akdasUAF/ForestFireDetection#16] [Q&A] [answered]
Hello Team (@YaliWang2019),

I've reviewed the existing repository and noted the development of U-Net and DBN models and the inclusion of the IEEE FLAME Dataset. With the project's objective to develop from scratch, I'm seeking guidance on the following:

1. **Existing Work Utilization:** Should the current repository serve as a base for further development, or is the goal to start anew?
2. **Project Direction and Priorities:** What areas need innovation or improvement? Are there specific technologies or methods we're looking to implement?

Best,
Yash Kumar
[kyash03.com](https://www.kyash03.com)

**Comment by YaliWang2019** [comment, marked as answer]
Hi Yash, 

The current repo is a base for further development. We have done training models for detecting fires. The areas that need improvement are tracking the fires via the bounding boxes and calculating the box areas to evaluate the seriosity of the fires. 
We are also looking forward to seeing some new innovations or technologies for training new models on detecting fires. We are working on providing a fully annotated dataset based on the IEEE FLAME Dataset for training. These dataset files should be available before the GSoC coding section starts.

Thanks and best,
Yali Wang

**Reply by KYash03** [reply]
Alright, that sounds good. Thank you.
